_dev/js/back/src/index.stories.js

```diff
@@ -0,0 +1,14 @@
+import { storiesOf } from "@storybook/vue";
+import i18n from "./lib/i18n";
+import Menu from "./components/menu/menu.vue";
+export default { title: "Menu" };
+
+const story = storiesOf("Menu", module);
+
+story.add("Default", () => ({
+  components: { Menu },
+  template: `<menu />`,
+  i18n
+}));
+
+export const withEmoji = () => "<div>😀 😎 👍 💯</div>";
```

Comment:
Not used at all right ?

Reply:
I don't know what this file do 😇

Reply:
It's used for Storybook, rendering components inside Storybook, but this last line is from a boilerplate I think